include/share/windows_unicode_filenames.h

```diff
@@ -39,21 +39,24 @@
 #include <sys/utime.h>
```

Comment:
Looks good. Can you just change the title to be just "Fix msvc build after commit b917d45" and then add:
```
Closes: https://github.com/xiph/flac/issues/152
```
to the commit message body?

Reply:
Done.